python/pyspark/sql/functions.py

```diff
@@ -397,7 +420,7 @@ def input_file_name():
 
 @since(1.6)
 def isnan(col):
-    """An expression that returns true iff the column is NaN.
+    """An expression that returns true if the column is NaN.
```

Comment:
I think "iff" is the abbreviation for "if and only if". I don't think it is worth changing.